sbin/build.sh

```diff
@@ -224,9 +230,9 @@ runTheOpenJDKConfigureCommandAndUseThePrebuildConfigParams()
 buildOpenJDK()
 {
   if [[ "$OS_MACHINE" == "s390x" ]] || [[ "$OS_MACHINE" == "armv7l" ]] ; then
```

Comment:
The scripts need to run outside of Jenkins too.
To ensure our build process is transparent and reproducible, there should be as little as possible in the scripts that depends upon our specific build environment -- and certainly nothing from the build environment that results in modifications to the final binary.